apps/condo/domains/resident/access/DiscoverServiceConsumersService.js

```diff
@@ -6,9 +6,8 @@ const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFo
 async function canDiscoverServiceConsumers ({ authentication: { item: user } }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
-    if (user.isAdmin) return true
 
-    return false
+    return !!user.isAdmin
```

Comment:
So, we assume nobody can run it in prod, yep?

Reply:
This mutation runs from models' hooks.